test/acceptance/features/registerServiceConstraints.feature

```diff
@@ -49,6 +50,82 @@ Feature: Register a cloud service in Primaza cluster with constraints but withou
         And On Primaza Cluster "main", ServiceCatalog "dev" will contain RegisteredService "primaza-rsdb"
 
 
+    Scenario: Cloud Service Registration, no Healthcheck provided, ServiceCatalog exists and RegisteredService constraint changes
+        Given Primaza Cluster "main" is running
+        And   Worker Cluster "worker" for ClusterEnvironment "worker" is running
+        And   Clusters "main" and "worker" can communicate
+        And   On Primaza Cluster "main", Worker "worker"'s ClusterContext secret "primaza-kw" for ClusterEnvironment "worker" is published
+        And   On Primaza Cluster "main", Resource is created
+        """
+        apiVersion: primaza.io/v1alpha1
+        kind: ClusterEnvironment
+        metadata:
+            name: worker
+            namespace: primaza-system
+        spec:
+            environmentName: dev
+            clusterContextSecret: primaza-kw
+        ---
+        apiVersion: primaza.io/v1alpha1
+        kind: RegisteredService
+        metadata:
```

Comment:
l71-94 can be removed right

Reply:
The idea here is to set it first to match so the catalog has the item, but when we do an update changing only the constraint to not match then the code added to remove from catalog is exercised. If we remove this then it will not exercise the removing part completely. Notice there are other test cases handling the not matching scenario on the create of the resource. This one focuses on the update, se L96

Reply:
Ok makes sense